test/concepts/test_concepts_receiver.cpp

```diff
@@ -1,129 +1,148 @@
-/*
- * Copyright (c) Lucian Radu Teodorescu
- *
- * Licensed under the Apache License Version 2.0 with LLVM Exceptions
- * (the "License"); you may not use this file except in compliance with
- * the License. You may obtain a copy of the License at
- *
- *   https://llvm.org/LICENSE.txt
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+// /*
```

Comment:
This change looks like it's commenting out everything??

Reply:
Yes, temporary change while I've been monkeying with the concepts. The receiver concept tests all need to be rewritten, but that didn't seem like the highest priority at the moment.